src/components/fonts.scss

```
@@ -0,0 +1,13 @@
+@font-face {
+  font-family: "Unhinted Rounded M+ 1p";
+  src: url("../fonts/unhinted-rounded-mplus-1p-regular.woff2");
+
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "Unhinted Rounded M+ 1p";
+  src: url("../fonts/unhinted-rounded-mplus-1p-bold.woff2");
+
+  font-weight: bold;
+}
```

元が300,400,500,700だったので、同じものを登録するようにしみました！

```suggestion
@font-face {
  font-family: "Unhinted Rounded M+ 1p";
  src: url("../fonts/unhinted-rounded-mplus-1p-light.woff2");
  font-weight: 300;
}

@font-face {
  font-family: "Unhinted Rounded M+ 1p";
  src: url("../fonts/unhinted-rounded-mplus-1p-regular.woff2");
  font-weight: 400;
}

@font-face {
  font-family: "Unhinted Rounded M+ 1p";
  src: url("../fonts/unhinted-rounded-mplus-1p-medium.woff2");
  font-weight: 500;
}

@font-face {
  font-family: "Unhinted Rounded M+ 1p";
  src: url("../fonts/unhinted-rounded-mplus-1p-bold.woff2");
  font-weight: 700;
}
```